examples/go/fakemodem/config.go

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+)
+
+/*****************
+ * Configuration *
+ *****************/
+
+var DefaultPlanUpdateIntervalInMinutes = 5
+
+// ChannelConfig defines the configuration for a specific
+// communications channel.
+type ChannelConfig struct {
```

Comment:
If "Channel" means ChannelSet in StellarStation API, I slightly prefer using the consistent word, ChannelSet here and other places in the PR.